README.md

```diff
@@ -710,6 +710,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
 - ### **H**
 
   - [Haafizz Ismail](https://github.com/haafizzismail)
+  - [Hizqeel Nizamani](https://github.com/ezekielnizamani)
```

Comment:
Please add your name in line 748, after Himangshu Kalita.
Let us know if you need any help with that. 🙂

Reply:
Thank you I added my name on line 748. 
But I can't see the pr button on my forked project. why is that?